.github/workflows/ci-pullrequest-main.yml

```
@@ -151,4 +151,10 @@ jobs:
       uses: docker/setup-buildx-action@v2
     
     - name: Docker Build
-      run: docker image build src/MockBot.Api/
+      run: docker image build src/MockBot.Api/ -t mockbot
```

For consistency, could you update this please?

```suggestion
      run: docker image build -f src/MockBot.Api/Dockerfile ./src -t mockbot
```